app.js

```diff
@@ -0,0 +1,123 @@
+class ToDoList {
+    constructor() {
+        this.toDoTasks = [
+            // {
+            //     id: 1,
+            //     task: 'kupic 4 bulki',
+            //     completed: false,
+            // },
+            // {
+            //     id: 2,
+            //     task: 'przedluzyc abonament',
+            //     completed: false,
+            // },11`1
+            // {
+            //     id: 3,
+            //     task: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quasi amet incidunt, mollitia vero doloremque quisquam corporis veritatis eum dolore commodi esse iste ratione nemo. Cumque ad eaque quo. Sapiente, perspiciatis?',
+            //     completed: true,
+            // }
+        ];
+
+        document.getElementById('addTaskBtn').addEventListener('click', this.addTask);
+        this.inputAddTask = document.getElementById('addTaskInput');
+        this.taskSection = document.querySelector('.toDoTasks');
+
+        this.render();
```

Comment:
Zamiast wywoływać render w constructorze, lepiej jest to zrobić na obiekcie `toDo` (`toDo.render()` na samym koncu pliku). Wywoływanie funkcji w constructorze jest ogólnie niezalecane.